jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java

```diff
@@ -144,36 +144,41 @@ private Line findRow(List<Line> rows, String name, String columnName) {
             throw exception("Can't find %s row with column %s", name, columnName);
         return line;
     }
-    @JDIAction("Assert that '{name}' is sorted by ascending")
+    @JDIAction("Assert that '{name}' is sorted by '{0}' in ascending order")
     public A sortedByAsc(String columnName) {
         List<String> column = table().webColumn(columnName).values();
         for (int i = 1; i < column.size(); i++)
             if (!isSorted(column.get(i-1), column.get(i), true, false))
                 jdiAssert("Table is not by ascending at "+i+" row", Matchers.is(""));
         return (A)this;
     }
-    @JDIAction("Assert that '{name}' is sorted by descending")
+    @JDIAction("Assert that '{name}' is sorted by '{0}' in descending order")
     public A sortedByDesc(String columnName) {
         List<String> column = table().webColumn(columnName).values();
         for (int i = 1; i < column.size(); i++)
             if (!isSorted(column.get(i), column.get(i-1), true, false))
                 jdiAssert("Table is not by descending at "+i+" row", Matchers.is(""));
         return (A)this;
     }
-    @JDIAction("Assert that '{name}' is sorted by ascending")
+    @JDIAction("Assert that '{name}' is sorted by '{0}' in ascending order")
     public A sortedNumByAsc(String columnName) {
         List<String> column = map(table().webColumn(columnName).values(), el -> el.replaceAll("[^0-9.,]", ""));
         for (int i = 1; i < column.size(); i++)
             if (!isSorted(column.get(i-1), column.get(i), true, false))
                 jdiAssert("Table is not by ascending at "+i+" row", Matchers.is(""));
         return (A)this;
     }
-    @JDIAction("Assert that '{name}' is sorted by descending")
+    @JDIAction("Assert that '{name}' is sorted by '{0}' in descending order")
     public A sortedNumByDesc(String columnName) {
         List<String> column = map(table().webColumn(columnName).values(), el -> el.replaceAll("[^0-9.,+\\-]", ""));
         for (int i = 1; i < column.size(); i++)
             if (!isSorted(column.get(i), column.get(i-1), true, false))
                 jdiAssert("Table is not by descending at "+i+" row", Matchers.is(""));
         return (A)this;
     }
+    @JDIAction("Assert that '{name}' preview is correct")
+    public A preview(String preview) {
+        jdiAssert(table().preview(), Matchers.is(preview));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5524853)